regression/rt_gk_sheath_1x2v_p1.c

```diff
@@ -487,24 +617,27 @@ main(int argc, char **argv)
 
   // Field.
   struct gkyl_gyrokinetic_field field = {
-    .fem_parbc = GKYL_FEM_PARPROJ_NONE, 
+    .fem_parbc = GKYL_FEM_PARPROJ_NONE,
     .kperpSq = ctx.k_perp * ctx.k_perp,
   };
 
-  // GK app.
+  // Gyrokinetic app.
   struct gkyl_gk app_inp = {
     .name = "gk_sheath_1x2v_p1",
 
-    .cdim = ctx.cdim, .vdim = ctx.vdim,
-    .lower = { -0.5 * ctx.Lz},
-    .upper = { 0.5 * ctx.Lz},
-    .cells = { cells_x[0] },
-    .poly_order = 1,
+    .cdim = 1, .vdim = 2,
+    .lower = { -0.5 * ctx.Lz },
+    .upper = { 0.5 * ctx.Lz },
+    .cells = { NZ },
+
+    .poly_order = ctx.poly_order,
     .basis_type = app_args.basis_type,
+    .cfl_frac = ctx.cfl_frac,
 
     .geometry = {
       .geometry_id = GKYL_MAPC2P,
-      .world = { 0.0, 0.0 },
+      .world = { 0.0, 0.0, 0.0 },
```

Comment:
this doesn't make sense to me. One should only need to pass the world coordinates in the non-simulated dimensions, not all 3

Reply:
Oops, sorry - I think I added this to the Lua tests (and therefore to the C tests) as part of tracking down some Valgrind uninitialized memory error. It turned out not to be the problem. Will remove this, thanks!

Reply:
FYI as a reminder, I think I saw this in a few tests, maybe both C and Lua, in case you forget